embark.el

```diff
@@ -883,7 +894,7 @@ If NESTED is non-nil subkeymaps are not flattened."
                            (eq cmd #'embark-keymap-help))
                    collect (list name
                                  (cond
-                                  ((keymapp cmd) 'keymap)
+                                  ((and (not (symbolp cmd)))(keymapp cmd) 'keymap)
```

Comment:
I'm guessing you wanted the keymapp inside the and.

Reply:
Fixed.